### DISCUSSION TITLE
Stitching operations into one .graphql file?

### DISCUSSION THREAD

**Discussion by mmahalwy** [dotansimha/graphql-code-generator#5667] [Ideas] [answered]
My directory `src/graphql/operations` contains a large number of operation files (in `.graphql` format). Ideally, I'd like a plugin to stitch them together into one file.

**Comment by dotansimha** [comment, marked as answer]
@mmahalwy This should be fairly simple, just use `concatAst` and then `print` from `graphql` library, in a custom plugin: https://graphql-code-generator.com/docs/custom-codegen/index 

If it's working for you, feel free to send a PR! ;)
